### PR TITLE
AR7881: Dashboard Branding Update

### PR DESCRIPTION
**Pull request by muktanshumishra24**
# Pull Request Template

Resolves or continues [AR-7881](...).

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

[AR-7881]: https://team-1624093970686.atlassian.net/browse/AR-7881?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/assets/biconomy-logo.png
Comment (shrinathprabhu) [inline src/assets/biconomy-logo.png:1]: Can this logo be svg?

File: src/assets/dapp-fallback.svg
Comment (shrinathprabhu) [inline src/assets/dapp-fallback.svg:1]: Check if this is the intended generic fallback, or if they have anything generic for fallback, this doesn't look like a fallback

File: src/assets/iconography/bug.png
Comment (shrinathprabhu) [inline src/assets/iconography/bug.png:1]: svg

File: src/assets/dapp-fallback.svg
Comment (muktanshumishra24) [reply src/assets/dapp-fallback.svg:1]: This is under design team as a task.

File: src/components/AppHeader.vue [hunk 1]
```diff
@@ -245,22 +253,44 @@ header {
...
+  padding-bottom: 1rem;
+  color: var(--primary-black);
   white-space: nowrap;
   list-style: none;
 }
 
+.help-menu-router:hover {
+  filter: brightness(0) saturate(100%) invert(50%) sepia(32%) saturate(4510%)
+    hue-rotate(304deg) brightness(100%) contrast(103%);
+}
```
Comment (shrinathprabhu) [inline src/components/AppHeader.vue:277]: Where did we get this value from and why is this required?

File: src/components/FullScreenLoader.vue [hunk 2]
```diff
@@ -22,8 +22,8 @@ defineProps({
         height: 100%;
       "
     >
-      <v-circular-progress color="var(--primary)" size="6em" />
-      <h4 style="margin-top: 1em">{{ message }}</h4>
+      <v-circular-progress color="#fff" size="6em" />
+      <h4 style="margin-top: 1em; color: #fff">{{ message }}</h4>
```
Comment (shrinathprabhu) [inline src/components/FullScreenLoader.vue:26]: no inline styles, change to tailwind classes

File: src/assets/iconography/bug.png
Comment (muktanshumishra24) [reply src/assets/iconography/bug.png:1]: This is an external icon I used, design team will be adding an icon for this as well.

File: src/components/SearchBar.vue [hunk 3]
```diff
@@ -6,12 +6,12 @@ const emits = defineEmits(['search'])
 
 <template>
   <div
-    class="flex space-x-3 bg-[#313131] items-center w-[200px] p-2 rounded-md"
+    class="flex space-x-3 bg-liquidlight items-center w-[200px] p-2 rounded-md border-[1px] border-black"
```
Comment (shrinathprabhu) [inline src/components/SearchBar.vue:9]: `border-[1px]` is `border`

File: src/components/app-configure/CreateApp.vue [hunk 4]
```diff
@@ -84,7 +84,7 @@ function onChainSelect(_, option) {
   <VOverlay>
     <div class="fixed inset-0 flex justify-center items-center">
       <div
-        class="rounded-[10px] flex-col p-8 max-[768px]:p-4 bg-[#262626] max-w-[560px] min-w-[200px] w-[70%] max-h-[100vh] overflow-y-auto"
+        class="rounded-[10px] flex-col p-8 max-[768px]:p-4 bg-[#FFF] max-w-[560px] min-w-[200px] w-[70%] max-h-[100vh] overflow-y-auto"
```
Comment (shrinathprabhu) [inline src/components/app-configure/CreateApp.vue:87]: `bg-[#fff]` is `bg-white`

Comment (shrinathprabhu) [reply src/components/app-configure/CreateApp.vue:87]: `max-h-[100vh]` is `max-h-screen`

File: src/components/app-configure/auth/CustomVerifier.vue [hunk 5]
```diff
@@ -267,7 +267,7 @@ watch(
 </script>
 
 <template>
-  <div class="border-2 border-[#363636] bg-[#1F1F1F] rounded-xl space-y-5">
+  <div class="bg-[#FFF] rounded-xl space-y-5">
```
Comment (shrinathprabhu) [inline src/components/app-configure/auth/CustomVerifier.vue:270]: `bg-[#fff]` is `bg-white`

File: src/components/app-configure/auth/IAMProviders.vue [hunk 6]
```diff
@@ -144,23 +144,25 @@ const areRequiredFieldsFilled = computed(() => {
 </script>
 
 <template>
-  <div class="border-2 border-[#363636] bg-[#1F1F1F] rounded-xl space-y-7">
+  <div class="bg-[#FFF] rounded-xl space-y-7">
```
Comment (shrinathprabhu) [inline src/components/app-configure/auth/IAMProviders.vue:147]: `bg-[#fff]` is `bg-white`

File: src/components/app-configure/auth/SocialAuthProvider.vue [hunk 7]
```diff
@@ -165,23 +165,25 @@ socialAuthStore.$subscribe((mutation) => {
 </script>
 
 <template>
-  <div class="border-2 border-[#363636] bg-[#1F1F1F] rounded-xl space-y-7">
+  <div class="bg-[#FFF] rounded-xl space-y-7">
```
Comment (shrinathprabhu) [inline src/components/app-configure/auth/SocialAuthProvider.vue:168]: `bg-[#fff]` is `bg-white`

File: src/components/app-configure/chain-management/AppChainManagementForm.vue [hunk 8]
```diff
@@ -135,7 +135,7 @@ async function fetchChainIdUsingRPCUrl(rpcURL: string) {
   <VOverlay>
     <div class="h-full flex overflow-y-auto py-2">
       <div
-        class="border-[1px] border-[#363636] rounded-lg max-h-[600px] w-[330px] text-white p-4 space-y-5 bg-[#1F1F1F] m-auto"
+        class="border-[1px] border-[#363636] rounded-lg max-h-[600px] w-[330px] text-black p-4 space-y-5 bg-[#FFF] m-auto"
```
Comment (shrinathprabhu) [inline src/components/app-configure/chain-management/AppChainManagementForm.vue:138]: `bg-[#fff]` is `bg-white`

File: src/components/app-configure/chain-management/AppChainManagementList.vue [hunk 9]
```diff
@@ -136,32 +136,32 @@ function isGaslessSupport(chainId: number) {
 </script>
 
 <template>
-  <div v-if="areChainsEmpty" class="text-white">
+  <div v-if="areChainsEmpty" class="text-black">
     <p class="text-center p-3">No Chains Available</p>
   </div>
-  <div v-else class="rounded-md border-[1px] border-[#363636]">
+  <div v-else class="bg-[#fff] rounded-md border-[1px] border-liquidgrey">
```
Comment (shrinathprabhu) [inline src/components/app-configure/chain-management/AppChainManagementList.vue:142]: `bg-[#fff]` is `bg-white`

File: src/components/app-configure/chain-management/AppChainManagementList.vue [hunk 10]
```diff
@@ -221,7 +221,7 @@ function isGaslessSupport(chainId: number) {
                 <ul
                   ref="showRowOptions_menu"
                   open
-                  class="flex flex-col bg-[#1F1F1F] text-[#FFFFFF] rounded-md border-[1px] border-[#363636] p-2 space-y-1 relative"
+                  class="flex flex-col bg-[#FFF] text-black rounded-md p-2 space-y-1 relative"
```
Comment (shrinathprabhu) [inline src/components/app-configure/chain-management/AppChainManagementList.vue:224]: `bf-[#fff]` is `bg-white`

**Review by shrinathprabhu — APPROVED** [review]
